portlets/src/main/webapp/vue-app/engagement-center/components/challenges/ChallengeCard.vue

```diff
@@ -202,6 +181,9 @@ export default {
     },
     isMobile() {
       return this.$vuetify.breakpoint.smAndDown;
+    },
+    isChallengeInitialised() {
+      return this.noParticipationYet && this.isActiveChallenge;
```

Comment:
There is a misunderstood when checking the name of the variable comparing to its value. What this variable is for? Besides, by using the current value, we will not display the winners when the challenge is a past challenge @srenault-meeds WDYT ?

```suggestion
    isChallengeInitialised() {
      return this.noParticipationYet && this.isActiveChallenge;
```

Reply:
i'll check it

Reply:
done ! @boubaker  removed isChallengeInitialised, there's no need for it 
and now the winners are showing if it has particiapants only ! 
as confirmed by @srenault-meeds